bucket/ds4windows.json

```diff
@@ -28,10 +30,22 @@
         "    </Action>",
         "</Actions>",
         "'@ }",
+        "    @{'Auto Profiles.xml' = '<?xml version=\"1.0\" encoding=\"utf-8\"?><Programs />' }",
+        "    @{'ControllerConfigs.xml' = '<?xml version=\"1.0\" encoding=\"utf-8\"?><Controllers />' }",
+        "    @{'LinkedProfiles.xml' = '<?xml version=\"1.0\" encoding=\"utf-8\"?><LinkedControllers />' }",
+        "    @{'OutputSlots' = '<?xml version=\"1.0\" encoding=\"utf-8\"?><OutputSlots />' }",
+        "    @{'Profiles.xml' = '<?xml version=\"1.0\" encoding=\"utf-8\"?><Profile><CheckWhen>0</CheckWhen></Profile>' }",
         ")",
+        "",
+        "# Create default files",
         "foreach ($k in $conf.Keys) {",
-        "    if (-not (Test-Path \"$persist_dir\\$k\")) { Set-Content \"$dir\\$k\" $conf.$k -Encoding Ascii }",
+        "    if (!(Test-Path \"$persist_dir\\$k\")) { ",
+        "        Write-Host \"'$k' does not exist. Creating.\" -f Yellow",
```

Comment:
```suggestion
```
This text message is not needed.

I am personally fine with this, but since a great number of packages does that, and almost none of them shows a message like this, we should follow the conventions.